dev/FHIR3-0-2-MM201901-Cert/Medication-9-0-7/_reference/MPClient/AdministrationAgreement-mp9-38129f10-100a-11e5-b939-0800200c9a66.xml

```diff
@@ -17,7 +17,7 @@
                </tr>
                <tr>
                   <th>Gemaakt</th>
-                  <td>2007-12-08</td>
+                  <td>${DATE, T, D, +0}</td>
```

Comment:
There are several changes like this that don't seem to be in scope for this ticket. Please revert them or explain why they are in scope.